packages/scripts/flow-translator/rewrite-imports.js

```diff
@@ -37,18 +37,16 @@ async function rewriteImportsInFolder(
   outputDir /*: string*/,
 ) {
   await fs.mkdir(outputDir, { recursive: true });
-  const dirents = await fs.readdir(inputDir, { withFileTypes: true });
+  const dirents = await fs.readdir(inputDir, {
+    recursive: true,
+    withFileTypes: true,
+  });
   for (const dirent of dirents) {
-    // We only care about the `stylex` package for now which has
-    // a flat file tree.
-    if (dirent.isDirectory()) {
-      continue;
-    }
```

Comment:
This script would silently fail to rewrite imports when not using a flat file tree. Should be fixed now.